ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -34,15 +34,12 @@ if [[ "${CLUSTER_TYPE}" == "ibmcloud"* ]]; then
   IC_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
   export IC_API_KEY
 fi
-if [[ "${CLUSTER_TYPE}" == "vsphere" ]]; then
-    declare cloud_where_run
+if [[ "${CLUSTER_TYPE}" == "vsphere"* ]]; then
     # shellcheck source=/dev/null
     source "${SHARED_DIR}/vsphere_context.sh"
```

Comment:
I see IBMC-DEVQE vSphere certificate is added to /var/run/vsphere8-secrets/vcenter-certificate, so this 2 line can be removed, as well as https://github.com/openshift/release/blob/3d8c8931b08ff29b966146c576cb4b60d19d4866/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L395C9-L395C9.

```
    # shellcheck source=/dev/null
    source "${SHARED_DIR}/vsphere_context.sh"
```

Reply:
thanks @sgaoshang, updated